packages/twenty-front/src/pages/onboarding/ChooseYourPlan.tsx

```diff
@@ -84,6 +84,10 @@ const StyledLinkGroup = styled.div`
   }
 `;
 
+const StyledChooseYourPlanPlaceholder = styled.div`
+  height: 566px;
```

Comment:
see with Thomas if we want to have a fix width on Modals (if it's driven by content, this will not handle well loading states)